perfscale_regression_ci/scripts/performance/conc_jobs.sh

```diff
@@ -0,0 +1,90 @@
+#/!/bin/bash
+################################################
+## Auth=skordas@redhat.com qili@redhat.com
+## Desription: Script to verify time of job creation with configMaps
+## Polarion test case: OCP-22822 - Concurrent job creation with ConfigMaps
+## https://polarion.engineering.redhat.com/polarion/redirect/project/OSE/workitem?id=OCP-22822
+## Bug related: https://bugzilla.redhat.com/show_bug.cgi?id=1686503 
+## https://github.com/kubernetes/kubernetes/issues/74412#issue-413387234 
+## Cluster config: default
+## job config: perfscale_regerssion_ci/content/conc_jobs.yaml
+################################################ 
+
+export jobs_amount=(${PARAMETERS:-100 500 1000 2000})
+results_file=conc_jobs_result_$(date +%Y%m%d%H%M%S).out
+
+function create_jobs() {
+  for i in $(seq 1 $1); do
+    cat ../../content/conc_jobs.yaml | sed "s/%JOB_ID%/$i/g" | oc create -f -
+  done
+}
+
+function wait_for_completion() {
```

Comment:
Since we are doing some re-writing here it would be good if we could make a couple of common bash scripts with all the common functions like these to not have to rewrite them in every test case. 

Could have a wait_for_completion, wait_for_termination, delete_project  etc functions in 1 file 

Maybe could put all those in 1 file for now and could separate later if file becomes too big. All these functions would be able to pass the pod name or project name we want to search for. 

The more robust these functions are now, the easier I think it'll be going forward with writing new test cases.

Reply:
Good suggestion! Done.